perms/manager.go

```diff
@@ -49,8 +49,13 @@ func NewManager(withAllSubServers bool) (*Manager, error) {
 	permissions := make(map[string]map[string][]bakery.Op)
 	permissions[litPerms] = RequiredPermissions
 	permissions[lndPerms] = lnd.MainRPCServerPermissions()
-	for k, v := range whiteListedLNDMethods {
-		permissions[lndPerms][k] = v
+
+	for k := range whiteListedLitMethods {
```

Comment:
If we're using the `IsWhiteListedURL` is being used, do we still need this? I assume yes, just to still recognize the URI as a valid call?

Reply:
resolving as discussed offline :)